src/ralph/models/xapi/constants.py

```diff
@@ -21,7 +21,17 @@
     "http://id.tincanapi.com/verb/viewed"
 ]
 
+VERB_PLAYED_DISPLAY = Literal["played"]  # pylint:disable=invalid-name
+VERB_PLAYED_ID = Literal[  # pylint:disable=invalid-name
+    "https://w3id.org/xapi/video/verbs/played"
+]
+
 # xAPI extensions
 EXTENSION_SCHOOL_ID = "https://w3id.org/xapi/acrossx/extensions/school"
 EXTENSION_COURSE_ID = "http://adlnet.gov/expapi/activities/course"
 EXTENSION_MODULE_ID = "http://adlnet.gov/expapi/activities/module"
+
+# xAPI activity types
+ACTIVITY_TYPE_VIDEO = Literal[  # pylint:disable=invalid-name
+    "https://w3id.org/xapi/video/activity-type/video"
+]
```

Comment:
This point needs to be rediscussed. I would consider the activity type of an object definition necessary to implement to strengthen uniqueness of the model